_posts/2023-11-30-prevent-broker-scale-down-if-containing-paritition-replicas.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "Preventing Kafka broker scale down if partition replicas are present"
+date: 2023-12-11
+author: Shubham Rawat
+---
+
+Apache Kafka is a platform designed for scalability.
+You can always scale in or scale out your Kafka Clusters based on your use case.
+When scaling down a cluster, it's crucial to ensure that data across brokers is moved or copied throughout the cluster.
+If you forget to move out the partition replicas from the broker being removed, then Strimzi might not be able to roll the cluster properly due to availability issues i.e. replicas assigned to that broker will be always offline. 
+Another issue would be that you would suffer data loss.
```

Comment:
```suggestion
Another issue would be that you might suffer data loss.
```

Reply:
Thanks for another pass Jakub. Applied the change